README.md

```diff
@@ -88,6 +89,11 @@ Then you could run `apply` or `file-state`, passing in sops encrypted secret fil
 vault-auto-config file-state --input-dir <config dir> --secrets secrets.yaml
 ```
 
+Or passing in a plaintext values file for the same purpose
+```shell script
+vault-auto-config file-state --input-dir <config dir> --additional-values secrets.yaml.dec
```

Comment:
Any reason to not just call it `--values` / `-v`, since that's not a thing it already accepts

Reply:
hmm good point